backend/pkg/kubeconfig/watcher.go

```diff
@@ -95,7 +101,8 @@ func addFilesToWatcher(watcher *fsnotify.Watcher, paths []string) {
 		// if it isn't, add it to the watcher
 		err := watcher.Add(path)
 		if err != nil {
-			log.Printf("Couldn't add %s to watcher: %v", path, err)
+			utils.Log(utils.LevelError, map[string]string{"path": path},
+				err, "Error adding path to watcher")
```

Comment:
I guess there needs to be some consistency in the logs with regards to "Error" being mentioned in the message. 

I expect "Error" should not be used, since it's removed from some of the messages?